tests/e2e/support/objects/app-files/resource/actions.ts

```diff
@@ -350,3 +354,25 @@ export const downloadResourceVersion = async (
   downloads.push(download)
   return downloads
 }
+
+export const emptyTrashBinResources = async (page): Promise<string> => {
+  await page.locator(emptyTrashBinButton).click()
+  await page.locator(actionConfirmButton).click()
+  const message = await page.locator(notificationMessageDialog).textContent()
+  return message.trim().toLowerCase()
+}
+
+export interface deleteResourceTrashbinArgs {
+  resource: string[]
+  actionType: string
+  page: Page
+}
+
+export const deleteResourceTrashbin = async (args: deleteResourceTrashbinArgs): Promise<string> => {
+  const { page, resource } = args
+  await selectOrDeselectResources({ page: page, names: resource, select: true })
```

Comment:
This function is causing `And "Carol" should not be able to delete following resources from the trashbin` this step to fail intermittently in CI and is failing most of the time locally. For trashbin we have a `path` selector which can directly check and uncheck the resource to delete.

Reply:
Updated same function to use a different selector for the trashbin. lmk if this looks good
https://github.com/owncloud/web/blob/91fda2047e6c32bbe2090f9349ed4ee444f85e4b/tests/e2e/support/objects/app-files/resource/actions.ts#L192-L194

Reply:
changed it back again so it selection is in the actual function itself 
https://github.com/owncloud/web/blob/71f078b2315731f5c62e439314f370d4d9f3be5d/tests/e2e/support/objects/app-files/resource/actions.ts#L376-L378 
`selectOrDeselectResources` has `resourceExists` which checks for `data-test-resource-name`

Reply:
seems ok for me.